adapters/IronSource/IronSourceAdapter/GADMAdapterIronSource.m

```diff
@@ -106,21 +109,50 @@ - (void)presentInterstitialFromRootViewController:(UIViewController *)rootViewCo
 #pragma mark Admob Banner
 
 - (void)getBannerWithSize:(GADAdSize)adSize {
-  [self showBannersNotSupportedError];
+    id<GADMAdNetworkConnector> strongConnector = _connector;
```

Comment:
We should do the following:
- Add the load ad method for banner ad in `GADMediationAdapterIronSource`, like what we did for [rewarded ad](https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/IronSource/IronSourceAdapter/GADMediationAdapterIronSource.m#L104-L112). - Create a new class `GADMAdapterIronSourceBannerAd` that implements [`GADMediationBannerAd`](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMediationBannerAd) and corresponding IronSource banner delegate(s).
- Update this class to return `GADMediationAdapterIronSouce` class, see [rewarded ad implementation](https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceRewarded.m).

We should also refactor interstitial ad to use the new APIs.